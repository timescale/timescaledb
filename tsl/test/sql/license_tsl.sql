-- This file and its contents are licensed under the Timescale License.
-- Please see the included NOTICE for copyright information and
-- LICENSE-TIMESCALE for a copy of the license.

\c :TEST_DBNAME :ROLE_SUPERUSER;

SHOW timescaledb.license;
SELECT _timescaledb_functions.tsl_loaded();
