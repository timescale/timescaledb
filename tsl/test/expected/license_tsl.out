-- This file and its contents are licensed under the Timescale License.
-- Please see the included NOTICE for copyright information and
-- LICENSE-TIMESCALE for a copy of the license.
\c :TEST_DBNAME :ROLE_SUPERUSER;
SHOW timescaledb.license;
 timescaledb.license 
---------------------
 timescale

SELECT _timescaledb_functions.tsl_loaded();
 tsl_loaded 
------------
 t

